modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscriptionManager.java

```diff
@@ -632,7 +632,9 @@ public AndesSubscription getSubscriptionByProtocolChannel(UUID channelID) {
         if (subIterator.hasNext()) {
             return subIterator.next();
         } else {
-            log.warn("No subscription found for channel ID " + channelID);
+            if (log.isDebugEnabled()) {
+                log.debug("No subscription found for channel ID " + channelID);
```

Comment:
Yes, each time a subscriber is connected, this happens. Besides it's not the correct place to print the log. This is just a method which returns suscriptions by the channel id. The one who actually knows if it's a valid channel id is the upper layer calling this. WDYT?